fcrepo-webapp/src/main/resources/logback.xml

```diff
@@ -1,37 +1,38 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE configuration>
 <configuration>
-    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
-        <encoder>                                                           
-            <pattern>%p %d{HH:mm:ss.SSS} \(%c{0}\) %m%n</pattern>
-        </encoder>
-    </appender>
+  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+    <encoder>
+      <pattern>%p %d{HH:mm:ss.SSS} \(%c{0}\) %m%n</pattern>
+    </encoder>
+  </appender>
 
-  <logger name="org.fcrepo.auth" additivity="false" level="${fcrepo.log.auth:-INFO}">
+  <logger name="org.fcrepo.auth" level="${fcrepo.log.auth:-null}">
     <appender-ref ref="STDOUT"/>
   </logger>
-  <logger name="org.fcrepo.connector.file" additivity="false" level="${fcrepo.log.connector.file:-INFO}">
+  <logger name="org.fcrepo.connector.file" level="${fcrepo.log.connector.file:-null}">
     <appender-ref ref="STDOUT"/>
   </logger>
-  <logger name="org.fcrepo.http.api" additivity="false" level="${fcrepo.log.http.api:-INFO}">
+  <logger name="org.fcrepo.http.api" level="${fcrepo.log.http.api:-null}">
     <appender-ref ref="STDOUT"/>
   </logger>
-  <logger name="org.fcrepo.http.commons" additivity="false" level="${fcrepo.log.http.commons:-INFO}">
+  <logger name="org.fcrepo.http.commons" level="${fcrepo.log.http.commons:-null}">
     <appender-ref ref="STDOUT"/>
   </logger>
-  <logger name="org.fcrepo.jms" additivity="false" level="${fcrepo.log.jms:-INFO}">
+  <logger name="org.fcrepo.jms" level="${fcrepo.log.jms:-null}">
     <appender-ref ref="STDOUT"/>
   </logger>
-  <logger name="org.fcrepo.kernel" additivity="false" level="${fcrepo.log.kernel:-INFO}">
+  <logger name="org.fcrepo.kernel" level="${fcrepo.log.kernel:-null}">
     <appender-ref ref="STDOUT"/>
   </logger>
-  <logger name="org.fcrepo.transform" additivity="false" level="${fcrepo.log.transform:-INFO}">
+  <logger name="org.fcrepo.transform" level="${fcrepo.log.transform:-null}">
     <appender-ref ref="STDOUT"/>
   </logger>
-  <logger name="org.fcrepo" additivity="false" level="${fcrepo.log:-INFO}">
+  <logger name="org.fcrepo" level="${fcrepo.log:-INFO}">
```

Comment:
But what I'm saying is that, with this configuration, I get duplicate messages in the log, and that's not a good default. Below is from `catalina.out` when deploying Fedora and making a single `GET` request. (You see that I get messages in triplicate)

```
INFO 09:13:51.945 (ModeShapeRepositoryFactoryBean) Using repo config (classpath): file:/usr/local/apache-tomcat-7.0.42/webapps/fcrepo/WEB-INF/classes/config/file-simple/repository.json
INFO 09:13:51.945 (ModeShapeRepositoryFactoryBean) Using repo config (classpath): file:/usr/local/apache-tomcat-7.0.42/webapps/fcrepo/WEB-INF/classes/config/file-simple/repository.json
INFO 09:13:51.945 (ModeShapeRepositoryFactoryBean) Using repo config (classpath): file:/usr/local/apache-tomcat-7.0.42/webapps/fcrepo/WEB-INF/classes/config/file-simple/repository.json
INFO 09:13:51.949 (DefaultPropertiesLoader) Loading properties
INFO 09:13:51.949 (DefaultPropertiesLoader) Loading properties
INFO 09:13:51.949 (DefaultPropertiesLoader) Loading properties
INFO 09:13:51.950 (DefaultPropertiesLoader) com.arjuna.ats.arjuna.common.ObjectStoreEnvironmentBean.default.objectStoreDir = /Users/acoburn/Projects/fcrepo4-data/com.arjuna.ats.arjuna.common.ObjectStoreEnvironmentBean.default.objectStoreDir
INFO 09:13:51.950 (DefaultPropertiesLoader) com.arjuna.ats.arjuna.common.ObjectStoreEnvironmentBean.default.objectStoreDir = /Users/acoburn/Projects/fcrepo4-data/com.arjuna.ats.arjuna.common.ObjectStoreEnvironmentBean.default.objectStoreDir
INFO 09:13:51.950 (DefaultPropertiesLoader) com.arjuna.ats.arjuna.common.ObjectStoreEnvironmentBean.default.objectStoreDir = /Users/acoburn/Projects/fcrepo4-data/com.arjuna.ats.arjuna.common.ObjectStoreEnvironmentBean.default.objectStoreDir
INFO 09:13:51.950 (DefaultPropertiesLoader) com.arjuna.ats.arjuna.objectstore.objectStoreDir = /Users/acoburn/Projects/fcrepo4-data/com.arjuna.ats.arjuna.objectstore.objectStoreDir
INFO 09:13:51.950 (DefaultPropertiesLoader) com.arjuna.ats.arjuna.objectstore.objectStoreDir = /Users/acoburn/Projects/fcrepo4-data/com.arjuna.ats.arjuna.objectstore.objectStoreDir
INFO 09:13:51.950 (DefaultPropertiesLoader) com.arjuna.ats.arjuna.objectstore.objectStoreDir = /Users/acoburn/Projects/fcrepo4-data/com.arjuna.ats.arjuna.objectstore.objectStoreDir
INFO 09:13:51.951 (DefaultPropertiesLoader) fcrepo.object.directory = /Users/acoburn/Projects/fcrepo4-data/fcrepo.object.directory
INFO 09:13:51.951 (DefaultPropertiesLoader) fcrepo.object.directory = /Users/acoburn/Projects/fcrepo4-data/fcrepo.object.directory
INFO 09:13:51.951 (DefaultPropertiesLoader) fcrepo.object.directory = /Users/acoburn/Projects/fcrepo4-data/fcrepo.object.directory
INFO 09:13:51.951 (DefaultPropertiesLoader) fcrepo.binary.directory = /Users/acoburn/Projects/fcrepo4-data/fcrepo.binary.directory
INFO 09:13:51.951 (DefaultPropertiesLoader) fcrepo.binary.directory = /Users/acoburn/Projects/fcrepo4-data/fcrepo.binary.directory
INFO 09:13:51.951 (DefaultPropertiesLoader) fcrepo.binary.directory = /Users/acoburn/Projects/fcrepo4-data/fcrepo.binary.directory
INFO 09:13:51.951 (DefaultPropertiesLoader) fcrepo.modeshape.index.directory = /Users/acoburn/Projects/fcrepo4-data/fcrepo.modeshape.index.directory
INFO 09:13:51.951 (DefaultPropertiesLoader) fcrepo.modeshape.index.directory = /Users/acoburn/Projects/fcrepo4-data/fcrepo.modeshape.index.directory
INFO 09:13:51.951 (DefaultPropertiesLoader) fcrepo.modeshape.index.directory = /Users/acoburn/Projects/fcrepo4-data/fcrepo.modeshape.index.directory
INFO 09:13:51.951 (DefaultPropertiesLoader) fcrepo.activemq.directory = /Users/acoburn/Projects/fcrepo4-data/activemq
INFO 09:13:51.951 (DefaultPropertiesLoader) fcrepo.activemq.directory = /Users/acoburn/Projects/fcrepo4-data/activemq
INFO 09:13:51.951 (DefaultPropertiesLoader) fcrepo.activemq.directory = /Users/acoburn/Projects/fcrepo4-data/activemq
INFO 09:14:44.076 (FedoraLdp) GET resource ''
INFO 09:14:44.076 (FedoraLdp) GET resource ''
INFO 09:14:44.076 (FedoraLdp) GET resource ''
```